class.jetpack-gutenberg.php

```diff
@@ -801,4 +814,49 @@ public static function get_extensions_preset_for_variation( $preset_extensions_m
 
 		return $preset_extensions;
 	}
+
+
+	/**
+	 * Check if the block should be available on the site.
+	 *
+	 * @return bool
+	 */
+	public static function is_premium_wpcom_available() {
```

Comment:
Maybe we should make this a bit more generic and have a general "paid" block check where we would pass the minimum plan needed for availability? This way we could use this in the future for other blocks that may require a different plan, without having to create a new function just for that.

Reply:
Yes, I was wondering about how generic to make it and what to call it as well.